extra-dev/packages/coq-metacoq-checker/coq-metacoq-checker.8.8.dev/opam

```diff
@@ -36,6 +37,7 @@ relation along with a reference type-checker that is extracted to a pluging.
 This provides a command: `MetaCoq Check [global_reference]` that can be used
 to typecheck a Coq definition using the verified type-checker.
 """
-url {
-   src: "git+https://github.com/MetaCoq/metacoq.git#coq-8.8"
-}
+# url {
+#   src: "https://github.com/MetaCoq/metacoq/archive/v2.1-beta3.tar.gz"
+#   checksum: "md5=e81b8ecabef788a10337a39b095d54f3"
+# }
```

Comment:
How is this supposed to work?

Reply:
Sorry, I should have made a draft PR, this is not ready